packages/react-instantsearch-dom/src/components/__tests__/MenuSelect.js

```diff
@@ -46,6 +46,20 @@ describe('MenuSelect', () => {
     expect(tree).toMatchSnapshot();
   });
 
+  it('default menu select with custom id', () => {
+    const tree = renderer
+      .create(
+        <MenuSelect
+          id="menu-select"
+          refine={() => {}}
+          items={[]}
+          canRefine={false}
+        />
+      )
+      .toJSON();
+    expect(tree).toMatchSnapshot();
```

Comment:
the test should rather be `getById('menu-select')` checking if it exists and is the select item

Reply:
Just updated to match the testing style of #3068!